src/Compiler/pars.fsy

```diff
@@ -5912,12 +5912,6 @@ braceBarExpr:
 braceBarExprCore:
   | LBRACE_BAR recdExprCore bar_rbrace
      { let orig, flds = $2
-       let flds =
```

Comment:
Is this the right level for motivation here, i.e. unifying the parser code for anons vs records (and reflecting that in SyntaxTree as well) ?

Since the motivation for a refactoring should be to make future changes easier, I would also like to use @brianrourkeboll judgement (since spreading will do work across records and anons) if this change is a change that brings benefits.